languages/fsharp/exercises/concept/records/.docs/instructions.md

```diff
@@ -0,0 +1,107 @@
+In this exercise you're a big sports fan and you've just discovered a passion for NBA basketball. Being new to NBA basketbal, you're doing a deep dive into NBA history, keeping track of teams, coaches, their win/loss records and comparing them against each other.
+
+As you don't yet have a favorite team, you'll also be developing an algorithm to figure out whether to root for a particular team.
+
+You have seven tasks to help you develop your proprietary _root-for-a-team_ algorithm.
+
+### 1. Define the model
+
+Define the `Coach` record with the following two fields:
+
+- `Name`: the coach's name, of type `string`.
+- `FormerPlayer`: indicates if the coach was a former player, of type `bool`.
+
+Define the `Record` record with the following two fields:
```

Comment:
Maybe `Stats` instead of `Record` so as not to overload the word?

Reply:
Good catch. I couldn't come up with a better name.